tests_e2e/tests/lib/ssh_client.py

```diff
@@ -28,11 +28,11 @@
 
 
 class SshClient(object):
-    def __init__(self, ip_address: str, username: str, private_key_file: Path, port: int = 22):
-        self._ip_address: str = ip_address
-        self._username: str = username
-        self._private_key_file: Path = private_key_file
-        self._port: int = port
+    def __init__(self, ip_address: str, username: str, identity_file: Path, port: int = 22):
```

Comment:
"private_key_file" is LISA's terminology. The SSH command line uses "identity_file"